<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snake by Veteška</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Port serveru:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>IP serveru:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="IPAddress">
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>213.211.37.18</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="ServerPort">
      <property name="inputMask">
       <string>99999</string>
      </property>
      <property name="placeholderText">
       <string>47777</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="JoinButton">
      <property name="text">
       <string>Připojit se</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_New_game"/>
    <addaction name="action_rejoin"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Konec</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_New_game">
   <property name="text">
    <string>Nová hra (1 hráč)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_rejoin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Připojit se</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
